docs/ExternalResults.asciidoc

```diff
@@ -0,0 +1,130 @@
+
+[[installing]]
+= openQA test harness result processing
+:toc: left
+:toclevels: 6
+:author: openQA developers
+
+== Introduction
+:parser-formats: <<parser-formats,parsers>>
+:api-endpoint: <<webapi-endpoint,Web Api endpoint>>
+
+From time to time, a test developer might want to use openQA to execute a test
+suite from different test harness than openQA, but still use openQA to setup test
+scenarios and prepare the environment for a test suite run;  for this case
+openQA console tests have support for parsing and displaying that the output that
+comes from external test harnesses in a compatible format.
```

Comment:
the sentence reads weird. "tests have support for parsing and displaying that the output that comes…"?

Reply:
does `parsing results from external test harnesses and displaying them` sounds better?